### PR TITLE
Do not extend #extent-length in defining #extent-measure (#846).

### LINKED ISSUE #846: Make #extent-length and #extent-measure complementary.
... otherwise a content profile cannot prohibit all `<measure>` expressions but `<length>` values

### PR DESCRIPTION
**Pull request by skynavga**
Closes #846.

### CODE REVIEW

**Comment by palemieux** [comment]
Build at https://rawgit.com/w3c/ttml2/issue-0846-extent-measure-feature-build/index.html

File: spec/ttml2.xml [hunk 1]
```diff
@@ -20723,7 +20723,6 @@ attribute.</p>
 <code>#extent-measure</code> feature if it implements presentation semantic support for
 <loc href="#style-value-measure">&lt;measure&gt;</loc> values when used with the
 <loc href="#style-attribute-extent"><att>tts:extent</att></loc> attribute.</p>
-<p>The <code>#extent-measure</code> feature is a syntactic and semantic extension of the <code>#extent-length</code> feature.</p>
 </div3>
```
Comment (palemieux) [inline spec/ttml2.xml]: `#extent-measure` needs to exclude `<length>` values, otherwise it remains an extension of `#extent-length`

**Comment by skynavga** [comment]
@palemieux addressed your comment in https://github.com/w3c/ttml2/pull/849/commits/987253b2604ec57532f5cca6c73116d326d5b421

**Comment by skynavga** [comment]
Merged early per WG resolution.

**Pull request merged**